### ISSUE TITLE
TaiChi-HD results reproducibility

### ISSUE THREAD

**Issue by PallottaEnrico** [iva-mzsun/AR-Diffusion#5]
Hi, thanks for your precious work, it's very interesting.
I was trying to reproduce the results on the Taichi-HD dataset but I'm struggling to reproduce the numbers in the paper (FIDimg ~13 and FVD16 ~66).

Here's the setup I've used:
- dataset downloaded from https://huggingface.co/datasets/maxin-cn/Taichi-HD/tree/main
- videos in this dataset are provided as sequence of pngs so I had to write my own dataloader changing just a bit VideoDataset, i.e. the get_frames function:
```     
        frames = [cv2.imread(frames_paths[i])[:, :, ::-1] for i in sample_idx]
        frames = np.array(frames)
        frames = torch.from_numpy(frames.transpose(0, 3, 1, 2))
        frames = frames.to(torch.float32) / 255.0  # NOTE: The image tensors are in [0, 1] for 1d tokenizer
        frames = self.transform(frames)
```
- set ARDIFF_STEP=50 as stated in the paper
- number of generated samples default to 256. Taichi test data consists of 285 videos in total
- used all the other config files as they are (paths adapted to my taichi dataset)
- used the ARDiff and ARVAE pretrained models on Taichi

I've run `stage2_ardiff_scripts/taichi_gen/infer_base_script.sh` twice, first using as valid data the test set of Taichi and then using as valid data the training set just to be sure.
With the test set I've got:
`{"FID-Img": 115.8771784404949, "FVD-3DRN50": 54.73947078233377, "FVD-3DInception": 543.1308200188919}`
With the training set:
`{"FID-Img": 64.18736818568578, "FVD-3DRN50": 24.342681638313707, "FVD-3DInception": 322.7537221981902}`
However both still far from the results reported in the paper.

Can you provide more details on the setting you've used to get those results?
Maybe I should be using an higher number of samples or there's a discrepancy between the dataset I've downloaded an the one you used.

Thanks for your patience.
